src/components/GraphFilter/DurationDropdown.tsx

```diff
@@ -13,6 +13,7 @@ type DurationDropdownState = {
 };
 
 export class DurationDropdown extends React.Component<DurationDropdownProps, DurationDropdownState> {
+  // TODO: use enum for type-safety and to make sure we always initialize this component with a valid value
```

Comment:
There is a jira for this already, can you update the comment to:
`// TODO: [KIALI-437] use enum for type-safety and to make sure we always initialize this component with a valid value`